liferay-echo/liferay-portal
# LPS-137270 buildLang

## What is inside this pr?
This PR adds more information in the collection filter modal's toolbar. Before this PR the message said "There are x results using the current filter". After this pr, the message include the label of all filter applied.

<div align="center">
  <img width="460" height="300" src="https://user-images.githubusercontent.com/11654230/129314195-fb503eb6-6ae3-4c70-90c9-149bd3adc8c8.png">
<img width="460" height="300" src="https://user-images.githubusercontent.com/11654230/129314200-d3d64580-466f-4ae0-a101-a1abfa316eff.png">
</div>

Figma link: https://www.figma.com/file/BwHy6HMAqPA0q8l5qXEMDW/LPS-126003-prefilter-the-collection?node-id=426%3A62026

## How to test this?

1. Deploy `layout-content-page-editor-display-context`.
2. Execute scr:enable com.liferay.journal.web.internal.info.collection.provider.BasicWebContentSingleFormVariationInfoCollectionProvider in gogo shell.
3. Add a Collection Display to a content page.
4. Select Basic Web Content collection provider.
5. A Filter Collection option will appear in the collection selector dropdown (it should be hidden for any other collection).
6. Introduce something in the filters
